packages/terra-form-textarea/src/terra-dev-site/doc/example/Resizable.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import Textarea from 'terra-form-textarea/lib/Textarea';
+
+const ResizableExample = () => (
+  <Textarea
+    isAutoResizable
+    size="small"
+    value="I automatically resize as you type in more information (except in mobile)."
```

Comment:
I am unable to type more text into this example on the deployed site

Reply:
yeah. Use that here instead.

Reply:
303ac6e